apps/design-system-docs/src/components/link-card/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Card, CardBody, CardMedia } from '@wordpress/components';
```

Comment:
Yay, we used a DS component in the doc site!

Reply:
Good old card